### PR TITLE
[Workers AI] Adds Gamertown demo

### PR DESCRIPTION
**Pull request by craigsdennis**
### Summary

Adds new external resources and demo apps

### CODE REVIEW

File: src/content/apps/index.yaml [hunk 1]
```diff
@@ -1,327 +1,335 @@
...
-  updated: 2024-08-29
-- link: https://github.com/ra-jeev/hub-chat
-  name: AI Chat
-  description: A full-stack application made with Nuxt to chat with various Cloudflare Workers AI LLM.
-  tags: [Nuxt]
-  products: [Pages, Workers AI]
-  languages: [TypeScript]
-  cloudflare: false
-  updated: 2024-08-29
+  - link: https://github.com/craigsdennis/phoney-ai
```
Comment (craigsdennis) [inline src/content/apps/index.yaml:2]: @kodster28 Why? On save this is autocorrecting to indented and adding a newline. I think it's using .editorconfig as intended but 🤷‍♂️

File: src/content/videos/index.yaml [hunk 2]
```diff
@@ -1,158 +1,171 @@
...
+    content_type: 🎥 Video
+    pcx_content_type: tutorial
+  - link: https://youtu.be/9IjfyBJsJRQ
+    title: Use Vectorize to add additional context to your AI Applications through RAG
+    description: A RAG based AI Chat app that uses Vectorize to access video game data for employees of Gamertown.
+    tags: [Workers, AI, Vectorize, RAG, Hono]
+    languages: [TypeScript]
+    products: [Workers, Workers AI, Vectorize]
+    cloudflare: true
+    # stream_id: 0b251ede0256f04f24f9127587e0c3aa
```
Comment (craigsdennis) [inline src/content/videos/index.yaml:166]: @kodster28 How do I get one of these?

Comment (kodster28) [reply src/content/videos/index.yaml:166]: For that, reach out to Jason F internally.

File: src/content/apps/index.yaml (hunk 1 above)
Comment (kodster28) [reply src/content/apps/index.yaml:2]: Yeah, it's pulling in the default `.editorconfig`. Painful in the short term, but should be nice once we hit all the files.

**Review by kodster28 — APPROVED** [review]
GTG from my end, @craigsdennis

**Pull request merged**